python/py-numpy/Portfile

```diff
@@ -11,14 +11,15 @@ categories-append       math
 license                 BSD
 maintainers             {michaelld @michaelld} openmaintainer
 description             The core utilities for the scientific library scipy for Python
-long_description        ${description}
+long_description        {*}${description}
 
-github.setup            numpy numpy 1.24.1 v
-checksums               rmd160  dd2df59cb48926eb460143f617a552b8009af11d \
-                        sha256  2386da9a471cc00a1f47845e27d916d5ec5346ae9696e01a8a34760858fe9dd2 \
-                        size    10903184
+github.setup            numpy numpy 1.25.2 v
```

Comment:
this version does no longer support Python 3.8 so you'll need to pin that subport to version 1.24.1

Reply:
Thanks, done.